src/aktualizr.cc

```diff
@@ -40,18 +40,23 @@ Aktualizr::Aktualizr(const Config &config) : config_(config) {
 #endif
 }
 
+Aktualizr::~Aktualizr() {
+  CRYPTO_cleanup_all_ex_data();
```

Comment:
I'm struggling to find good documentation on this function, but what happens if someone creates two Aktualizr objects in the same application?

Would curl_global_cleanup (to match curl_global_init) fix the problem?